apps/condo/domains/billing/schema/DiscoverServiceConsumersService.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Generated by `createservice billing.DiscoverServiceConsumersService --type mutations`
+ */
+
+const { get } = require('lodash')
+
+const { GQLCustomSchema, getById } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/billing/access/DiscoverServiceConsumersService')
+const { BillingAccount } = require('@condo/domains/billing/utils/serverSchema')
+const { Resident, ServiceConsumer } = require('@condo/domains/resident/utils/serverSchema')
+
+const DiscoverServiceConsumersService = new GQLCustomSchema('DiscoverServiceConsumersService', {
+    types: [
+        {
+            access: true,
+            type: 'input DiscoverConsumersServiceInput { dv: Int!, sender: SenderFieldInput!, address: String!, unitName: String!, unitType: String!, billingAccountId: ID, residentId: ID }',
+        },
+    ],
+    
+    mutations: [
+        {
+            access: access.canDiscoverConsumersService,
+            schema: 'discoverConsumersService(data: DiscoverConsumersServiceInput!): [ServiceConsumer]',
+            resolver: async (parent, args, context) => {
+                const { data } = args
+                const { address, unitName, unitType, billingAccountId, residentId } = data
+
+                const residents = residentId ? await Resident.getAll(context, { id: residentId, deletedAt: null }) : await Resident.getAll(context, {
+                    address: address,
+                    unitName: unitName,
+                    unitType: unitType,
+                    deletedAt: null,
+                })
+
+                const billingAccounts = billingAccountId ? await BillingAccount.getAll(context, { id: billingAccountId, deletedAt: null })
```

Comment:
Why not no other filters there?

Reply:
We have 3 use cases here
1. On creation of new BillingAccount -> auto bind this account to all residents
2. On creation of new Resident -> auto bind all existing accounts to this resident
3. Manual run to bind all existing residents to all existing accounts for address

So, there will be no other filters